handbook/product-design/README.md

```diff
@@ -37,9 +37,11 @@ If a customer/prospect request is missing a Gong snippet or requires additional
 As soon as we've addressed the next quarter's objectives, the Head of Product Design cancels the daily meeting. 
 
 
-### Product design check in
+### Triage new bugs
 
-The Head of Product Design summarizes the current week's design reviews to discuss with the CEO.
+Product Designers are responsible for triaging all new bugs.
```

Comment:
```suggestion
Product Designers are responsible for triaging all new reproduced bugs.
```

Reply:
Unless you think they should triage initial bug reports that still need to be reproduced?

Reply:
Good catch. I think let's keep this to "reproduced bugs" for now.